txs/conservative_state.go

```diff
@@ -211,8 +209,8 @@ func (cs *ConservativeState) getTXsToApply(logger log.Log, toApply *types.Block)
 				return nil, fmt.Errorf("applying block %s with invalid tx %s", toApply.ID(), mtx.ID)
 			}
 			mtx.TxHeader = header
-			// updating header also updates principal/nonce indexes
-			if err = transactions.AddHeader(cs.db, mtx.ID, header); err != nil {
+			// add updates header
```

Comment:
Keeping the case the same provides better comment searchability in jetbrains
```suggestion
			// Add updates header
```